kevm_pyk/src/kevm_pyk/__main__.py

```diff
@@ -19,6 +19,12 @@
 _LOG_FORMAT: Final = '%(levelname)s %(asctime)s %(name)s - %(message)s'
 
 
+def _ignoring_arg(args: Dict[str, Any], arg: str, cli_option: str):
+    if arg in args and args[arg]:
+        _LOGGER.warning(f'Ignoring command-line option: {cli_option}')
+    args.pop(arg)
```

Comment:
If `arg not in args`, statement `args.pop(arg)` raises `KeyError`.

If `args[arg]` is falsy, the log message will not be emitted. 

```suggestion
def _ignore_arg(args: Dict[str, Any], arg: str, cli_option: str) -> None:
    if arg in args:
        if args[arg] is not None:
            _LOGGER.warning(f'Ignoring command-line option: {cli_option}')
        args.pop(arg)
```